src/modules/edc-demo/components/policy-view/policy-view.component.html

```diff
@@ -33,17 +33,32 @@
 
     <!-- Paginator -->
     <mat-paginator
+      *ngIf="policyList.isReady"
       hidePageSize="true"
       pageSize="10"
-      [length]="policyDefinitions?.length"></mat-paginator>
+      [length]="policyList.data.filteredPolicies.length"></mat-paginator>
   </div>
 
-  <div fxLayout="column" fxLayoutAlign="space-around stretch">
-    <mat-card *ngFor="let policyDef of policyDefinitions" class="policy-card">
+  <div class="flex flex-row justify-center uppercase">
```

Comment:
The uppercase class here is right now making empty messages uppercase.

It should be removed here and empty state / error state should specify such styling.